.evergreen/functions.yml

```diff
@@ -669,12 +669,29 @@ functions:
           # Load environment variables
           eval $(.evergreen/print-compass-env.sh)
 
+          if [[ "$IS_WINDOWS" == "true" ]]; then
+            // TODO: windows_setup
```

Comment:
We could also just call the smoketest script for each of these TODOs now already because it will only be executed for mac. We'd just have to leave osx_zip out for now.